### ISSUE TITLE
Refractory Soaking Pot

### ISSUE THREAD

**Issue by Chenjox** [codetaylor/pyrotech#293]
It would be nice If one could have a soaking pot that doesn't break at temp > 450°.
And of course having ct integration.

**Comment by Derpercoaster** [comment]
I do not think we actually need one. The only "hot" liquid in the game is lava, and that really would serve no purpose, since there is nothing that you can soak in it.

**Comment by Birgliss** [comment]
This is primarily for fluids added by other mods, and for recipes created by pack developers, I feel. 

Maybe it won't see a use in Pyrotech itself, but having it available wouldn't hurt.

**Comment by codetaylor** [comment]
I don't intend to add stuff to Pyrotech that doesn't have a use in a Vanilla + Pyrotech context. If the mod evolves to have hot fluids for soaking, then I'll consider it. Adding stuff that is only useful if configured with other mods isn't part of Pyrotech's design philosophy.

**Comment by Birgliss** [comment]
In terms of actual features, "soaking in lava" reminds me of the scorched bricks from Mariculture, which is pretty much the only reason I spoke up on this issue at all, lol. Could be a way to craft nether bricks, close #240 at the same time as this one. Just spitballing.

**Comment by codetaylor** [comment]
> Sorry if I'm stepping on your toes here

Don't worry, you're not. :)

That's not a bad idea and I'll take it into consideration, thank you.

**Comment by hron84** [comment]
I could also imagine clay brick => nether brick conversion by lava. Currently I trying to integrate Primal Core and Pyrotech because the very interesting cross-mod potentials (crafting mechanics, initial in-world age, etc) and it would fit very well.

**Comment by Rsslone** [comment]
In my player base testing, they found the soaking pot to suck at a certain point and was asking about an upgraded version. I don't really think these hot fluids matter to my players, if it just had the same fueling and auto stopping features the refractory machines do, this would be a huge addition.

For now to make my pack less grindy feeling at the refractory stage of the game, I'll be removing the fire reqs from the living tar recipe.

**Comment by GManMan** [comment]
> In my player base testing, they found the soaking pot to suck at a certain point and was asking about an upgraded version. I don't really think these hot fluids matter to my players, if it just had the same fueling and auto stopping features the refractory machines do, this would be a huge addition.
> 
> For now to make my pack less grindy feeling at the refractory stage of the game, I'll be removing the fire reqs from the living tar recipe.

A brick/refractory style of the soaking pot then?

**Comment by Rsslone** [comment]
> A brick/refractory style of the soaking pot then?

Exactly.
